src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] **Required:** Kindly follow the ducks pattern as stated previously in https://github.com/jsug9/bookstore/blob/22f5de3f6d0b337c2fd0c3e44c13f894eff1e79f/src/redux/books/books.js#L1-L2

Reply:
Dear @brytebee thank you for the suggestions, I'll fix it right now.